webapp/channels/src/components/advanced_text_editor/send_button/send_post_options/core_menu_options.test.tsx

```diff
@@ -0,0 +1,180 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {DateTime} from 'luxon';
+import React from 'react';
+
+import {renderWithContext, fireEvent, screen} from 'tests/react_testing_utils';
+
+import CoreMenuOptions from './core_menu_options';
+
+jest.mock('components/advanced_text_editor/use_post_box_indicator', () => ({
```

Comment:
solved in my last commit. TS was enforcing to return all the values from the cstom hook, and to avoid creating a type and use a partial, I just did what TS was demanding on top of the file and destructure on every call and modify a value as required for the test.